src/libsync/configfile.h

```diff
@@ -256,6 +256,14 @@ class OWNCLOUDSYNC_EXPORT ConfigFile
     [[nodiscard]] static QString discoveredLegacyConfigPath();
     static void setDiscoveredLegacyConfigPath(const QString &discoveredLegacyConfigPath);
 
+    static const QString isVfsEnabledC;
+    static const QString launchOnSystemStartupC;
+    static const QString optionalServerNotificationsC;
+    static const QString promptDeleteC;
+    static const QString showCallNotificationsC;
+    static const QString showChatNotificationsC;
+    static const QString showInExplorerNavigationPaneC;
```

Comment:
you should probably still use the same data type especially for static class attirbutes
can you test with `constexpr char <attribute name>[]` ?